app/javascript/app/components/sdg-card/sdg-card-component.jsx

```diff
@@ -58,11 +59,25 @@ class SDGCard extends PureComponent {
                 targetData.targets[target.number].sectors;
               return (
                 <span
```

Comment:
We are using a span with a javascript event to go to another page... it makes sense in a map for example, but in a static markup element we could use directly an <a> element, couldn't we?
Also is much better for accessibility, so could we move this to a `Link` component from react-router?